### ISSUE TITLE
`policy_loss` in `04_PG`

### ISSUE THREAD

**Issue by BellaCoola** [alessiodm/drl-zh#10]
Hello, sorry in advance for maybe a stupid question. I'd like to ask why is the `policy_loss` in `04_PG` formulated as `policy_loss = -(log_probs * returns).sum()`? Maybe I missed something but in the Jupyter Notebook there is no mention on how to compute the policy loss itself. There are only descriptions how to update network's weights and a reference to [Hugging Face Deep RL tutorial](https://huggingface.co/learn/deep-rl-course/unit4/pg-theorem), which mentions the following objective function, however that objective function does not contain a log() term.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/f47e395b-f914-4098-b567-0b4ebfdc4fdc" />

**Comment by alessiodm** [comment]
Thank you @BellaCoola 🙏  It is really great to hear when this course is useful and provides some value.  Please, let me know how the rest of the experience with the following notebooks goes!
